.idea/runConfigurations/Application.xml

```diff
@@ -1,22 +1,20 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="Application" type="Application" factoryName="Application" nameIsGenerated="true">
-    <extension name="coverage" enabled="false" merge="false" sample_coverage="true" runner="idea">
+    <envs>
+      <env name="API_TOKEN" value="sk_df95b4285f3144908348526c7e2a2215" />
```

Comment:
This is interesting. You're providing the API token as an environment variable in the run configuration. I don't think this is the best place for it. The run configurations should be a very basic setup designed to provide easy access to the _basic_ takss any developer may want to run. We would very rarely customize them or change them after creating them. There are other ways we can provide the api key without modifying the run configurations, however we should also keep in mind the second point...

We never want to commit any private (passwords, api keys, tokens, etc.) information that's used for authentication/authorization. Source control is never the proper place to store that information. What you _can_ do if you don't want to provide it each time your application starts up, is put it in a file that's used in your application (say, secrets.yml), but also add that file to the .gitignore file. Git will then ignore that file when committing code

Reply:
I moved the API key to a separate file that I added to .gitignore